apiserver/common/tools.go

```diff
@@ -100,6 +124,47 @@ func NewToolsGetter(
 	}
 }
 
+// GetEntityAgentVersion is responsible for getting the target agent version for
+// a given tag.
+func (t *ToolsGetter) GetEntityAgentVersion(
```

Comment:
Doesn't need to be exported

Also, we're trying to get rid of these embedded apiserver facade structs.

Instead, just copy the ` Tools(ctx context.Context, args params.Entities) (params.ToolsResults, error)` method to each of the 3 facades that embeds it and call the relevant service method. Yes, it's a small amount of code duplication, but it allows facade versioning etc. We've run into problems in the past where this has been an issue and we can now address it. Once done,we can delete the `common/tools.go`

Reply:
Have changed the export. Can we come back and do the move work later? Just trying to get model config done at the moment.